*** tests/a-11-14-m-ok.out	2019-12-10 17:49:09.684723200 +0000
--- tests/a-11-14-m-ok.outhyp	2019-12-10 18:05:26.956905800 +0000
***************
*** 29,35 ****
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Introduza o número da obra: A obra 1 deve ser devolvida no dia 22.
  Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
--- 29,35 ----
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Introduza o número da obra: A obra 1 deve ser devolvida no dia 28.
  Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
