*** tests/a-13-03-m-ok.out	2019-12-10 17:49:19.748873700 +0000
--- tests/a-13-03-m-ok.outhyp	2019-12-10 18:05:44.103025700 +0000
***************
*** 78,84 ****
  4 - Mostrar notificações de utente
  5 - Saldar multa de utente
  0 - Sair
! Escolha uma opção: Introduza o número de utente: 2 - Rey - real.last.jedi@universe.com - NORMAL - ACTIVO
  Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
--- 78,84 ----
  4 - Mostrar notificações de utente
  5 - Saldar multa de utente
  0 - Sair
! Escolha uma opção: Introduza o número de utente: 2 - Rey - real.last.jedi@universe.com - NORMAL - SUSPENSO - EUR 110
  Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
