*** tests/a-13-01-m-ok.out	2019-12-06 16:49:54.000000000 +0000
--- tests/a-13-01-m-ok.outhyp	2019-12-08 15:51:55.000000000 +0000
***************
*** 89,96 ****
  5 - Saldar multa de utente
  0 - Sair
  Escolha uma opção: 0 - Chewbacca - chewbacca@milenium.falcon.org - NORMAL - ACTIVO
! 1 - Darth Vader - darth@dark.force.coAm - FALTOSO - SUSPENSO - EUR 90
! 2 - Luke Skywalker - last.jedi@universe.com - NORMAL - SUSPENSO - EUR 25
  Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
--- 89,96 ----
  5 - Saldar multa de utente
  0 - Sair
  Escolha uma opção: 0 - Chewbacca - chewbacca@milenium.falcon.org - NORMAL - ACTIVO
! 1 - Darth Vader - darth@dark.force.coAm - FALTOSO - ACTIVO
! 2 - Luke Skywalker - last.jedi@universe.com - NORMAL - ACTIVO
  Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
***************
*** 98,104 ****
  4 - Mostrar notificações de utente
  5 - Saldar multa de utente
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
  3 - Listar utentes
--- 98,105 ----
  4 - Mostrar notificações de utente
  5 - Saldar multa de utente
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Saldar multa de utente: Operação inválida: O utente 1 não se encontra suspenso.
! Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
  3 - Listar utentes
***************
*** 107,113 ****
  0 - Sair
  Escolha uma opção: 0 - Chewbacca - chewbacca@milenium.falcon.org - NORMAL - ACTIVO
  1 - Darth Vader - darth@dark.force.coAm - FALTOSO - ACTIVO
! 2 - Luke Skywalker - last.jedi@universe.com - NORMAL - SUSPENSO - EUR 25
  Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
--- 108,114 ----
  0 - Sair
  Escolha uma opção: 0 - Chewbacca - chewbacca@milenium.falcon.org - NORMAL - ACTIVO
  1 - Darth Vader - darth@dark.force.coAm - FALTOSO - ACTIVO
! 2 - Luke Skywalker - last.jedi@universe.com - NORMAL - ACTIVO
  Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
***************
*** 115,121 ****
  4 - Mostrar notificações de utente
  5 - Saldar multa de utente
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
  3 - Listar utentes
--- 116,123 ----
  4 - Mostrar notificações de utente
  5 - Saldar multa de utente
  0 - Sair
! Escolha uma opção: Introduza o número de utente: Saldar multa de utente: Operação inválida: O utente 2 não se encontra suspenso.
! Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
  3 - Listar utentes
